crates/tabby/Cargo.toml

```diff
@@ -24,7 +24,7 @@ hyper = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 utoipa = { workspace = true, features = ["axum_extras", "preserve_order"] }
 utoipa-swagger-ui = { version = "3.1", features = ["axum"] }
-serde = { workspace = true }
+serde = { workspace = true, features = ["rc"] }
```

Comment:
is this necessary?

Reply:
Not anymore, I had used it in a previous version but removed the rc that was stored inside a serialized struct